pennylane/ops/qubit/qchem_ops.py

```diff
@@ -678,17 +781,18 @@ class DoubleExcitationPlus(Operation):
     num_params = 1
     """int: Number of trainable parameters that the operator depends on."""
 
+    ndim_params = (0,)
+    """tuple[int]: Number of dimensions per trainable parameter that the operator depends on."""
+
     grad_method = "A"
     """Gradient computation method."""
 
     parameter_frequencies = [(1,)]
     """Frequencies of the operation parameter with respect to an expectation value."""
 
     def generator(self):
-        G = -1 * np.eye(16, dtype=np.complex64)
+        G = 1j * DoubleExcitation.mask_s - I16
```

Comment:
Why this is necessary?

Reply:
I thought this was a nice way of reusing predefined and precomputed objects, but it actually is not! :o
Will revert!

Reply:
I wonder, though, why we are using `complex64` instead of `complex128` here? :thinking:

Reply:
> I wonder, though, why we are using complex64 instead of complex128 here? 

I cannot think of any specific reason.

Reply:
I'll leave it as is, just because it is out of scope and seems to be working fine :)